### PR TITLE
added core-ml intern JD

### PR DESCRIPTION
**Pull request by sanchit-ahuja**
Added the intern JD. Mostly inspired from the core-ml engineer JD and some from Research fellow

### CODE REVIEW

File: machine-learning/core-ml-intern.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+---
+authors: @sanchit
+groups: product
+teams: core-ml
+level: NA
+active: true
+---
+# Machine Learning Research Intern
```
Comment (lepisma) [inline machine-learning/core-ml-intern.md]: ML Intern, Core ML

Comment (lepisma) [reply machine-learning/core-ml-intern.md]: Also let's change the name of file also

File: machine-learning/core-ml-intern.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+We are looking for an ML Research intern for a duration of 5 months. As an intern, you will be working closely with the core-ml team. You will be involved in owning various product initiatives, creating protypes and delving into research papers.
```
Comment (lepisma) [inline machine-learning/core-ml-intern.md]: name and duration

File: machine-learning/core-ml-intern.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+authors: @sanchit
+groups: product
+teams: core-ml
+level: NA
+active: true
+---
+# Machine Learning Research Intern
+
+We are looking for an ML Research intern for a duration of 5 months. As an intern, you will be working closely with the core-ml team. You will be involved in owning various product initiatives, creating protypes and delving into research papers. 
+Our team works closely with the research team, broadly on the topics such as Spoken Language Understanding, Speech Recognition, Paralinguistics and Response Production.
```
Comment (lepisma) [inline machine-learning/core-ml-intern.md]: We don't have to say the team, we can make it like the person will work closely with our product and research teams etc.

File: machine-learning/core-ml-intern.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+### Responsibilities
+
++ Design, build and evaluate Machine Learning systems.
++ Perform experiments and statistical analyses to draw conclusions and take
+  modeling decisions.
++ Study, implement and extend state of the art systems.
++ Write research outcomes and publish documentations
```
Comment (lepisma) [inline machine-learning/core-ml-intern.md]: We can remove this part. This intern will be more focused on product development work.

File: machine-learning/core-ml-intern.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
++ Take part in regular research reviews and discussions.
++ Build, maintain and extend our open source solutions in the domain.
++ Write well-crafted programs at all levels of the system. This includes the
```
Comment (lepisma) [inline machine-learning/core-ml-intern.md]: can remove this line .too much expectation from an intrern.

File: machine-learning/core-ml-intern.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+  and evaluation, visualization and monitoring systems.
+ 
+### Requirements
+
++ Practical Machine Learning experience as demonstrated by earlier works.
++ Knowledge of and ability to use tools from theoretical and practical aspects
+  of computer science. This includes, but is not limited to, probability,
+  statistics, learning theory, algorithms, software architecture, programming
+  languages, etc.
++ Good programming skills and ability to work with programs at all levels of a
```
Comment (lepisma) [inline machine-learning/core-ml-intern.md]: Can also be specific and ask for good 'prototyping skills' too

**Review by lepisma — CHANGES_REQUESTED** [review]

File: machine-learning/core-ml-intern.md (hunk 2 above)
Comment (sanchit-ahuja) [reply machine-learning/core-ml-intern.md]: Should we mention the duration (and also the tentative months) or is it flexible? Name as in? @lepisma

Comment (lepisma) [reply machine-learning/core-ml-intern.md]: yes, flexible. can put starting month as immediate.

Name shold be 'ML Intern' and not 'ML Research Intern'

**Review by lepisma — CHANGES_REQUESTED** [review]

File: machine-learning/ml-intern-core-ml.md [hunk 7]
```diff
@@ -0,0 +1,39 @@
+---
+authors: @sanchit
+groups: product
+teams: core-ml
+level: ML Intern, Core ML
```
Comment (lepisma) [inline machine-learning/ml-intern-core-ml.md]: Level wrong

File: machine-learning/ml-intern-core-ml.md [hunk 8]
```diff
@@ -0,0 +1,39 @@
...
+active: true
+---
+# Machine Learning Intern
+
+We are looking for an ML Research intern for a duration of 5 months to join immediately (duration and date of joining are flexible). As an intern, you will be working closely with the product and research team. You will be involved in owning various product initiatives, creating protypes and delving into research papers.
```
Comment (lepisma) [inline machine-learning/ml-intern-core-ml.md]: Let's do 2 to 5 months instead of 5 months?

**Review by lepisma — CHANGES_REQUESTED** [review]

File: machine-learning/ml-intern-core-ml.md (hunk 7 above)
Comment (sanchit-ahuja) [reply machine-learning/ml-intern-core-ml.md]: What level to put? L1 or leave it NA? @lepisma

Comment (lepisma) [reply machine-learning/ml-intern-core-ml.md]: NA

**Pull request merged**